Upstream: No
Reason: Fix default python version for this script

--- a/src/gitversion/getversion.py	2020-06-05 00:07:29.206108295 +0300
+++ b/src/gitversion/getversion.py	2020-06-05 00:07:48.846126599 +0300
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 import versioneer
 
--- a/src/gitversion/gitversion.cmake	2020-06-05 00:39:47.862225137 +0300
+++ b/src/gitversion/gitversion.cmake	2020-06-05 00:40:18.162282012 +0300
@@ -1,7 +1,7 @@
 set(DIR_OF_GITVERSION_TOOL "${CMAKE_CURRENT_LIST_DIR}" CACHE INTERNAL "DIR_OF_GITVERSION_TOOL")
 
 function (get_git_version OUTPUT_VARIABLE)
-    EXECUTE_PROCESS(COMMAND python ${DIR_OF_GITVERSION_TOOL}/getversion.py
+    EXECUTE_PROCESS(COMMAND python3 ${DIR_OF_GITVERSION_TOOL}/getversion.py
         WORKING_DIRECTORY ${DIR_OF_GITVERSION_TOOL}
         OUTPUT_VARIABLE VERSION
         ERROR_VARIABLE error
